src/app/test-record/visit/visit-edit/visit-edit.component.scss

```diff
@@ -0,0 +1,10 @@
+#test-testStationType {
+  border: none;
+}
+#test-testStationType:focus {
+  outline: none;
+  box-shadow: none;
+}
+#test-testStationType:disabled {
+  color: #0b0c0c;
```

Comment:
same for scss, we should not be targeting id, this make high specificity and makes it hard to override